### DISCUSSION TITLE
This type does not have a value, so it cannot be used as injection token

### DISCUSSION THREAD

**Discussion by tesshsu** [abacritt/angularx-social-login#550] [General]
I tried to integrate with the plugin follow by doc
I declare as follow
```

constructor ( private authService: SocialAuthService )
async loginGoogle() {
  let googleUser = this.authService.signIn(GoogleLoginProvider.PROVIDER_ID);
}

```
Thne I got 
private authService: SocialAuthService
                                ~~~~~~~~~~~~~~~~~
    This type does not have a value, so it cannot be used as injection token.

Any ideal how to reslove this issue ?

**Comment by mfrizqi** [comment]
As written in here [sign in with google](https://github.com/abacritt/angularx-social-login#sign-in-with-google
you dont need to write signIn for google as the method has replaced by asl button

and also can you add line where you add imports from SocialAuthService?
